modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -77,6 +77,31 @@ $ kubectl -n openshift-network-operator logs -l "name=network-operator"
 +
 On high availability clusters with three or more control plane (master) nodes, the Operator will perform leader election and all other Operators will sleep. For additional details, see https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md[Troubleshooting].
 
+== Addressing the `No disk found with matching rootDeviceHints` error
+
+After the deployment of a cluster you might receive the following error:
+
+----
+`No disk found with matching rootDeviceHints`
+----
+
+To address the "No disk found with matching rootDeviceHints" error, a temporary workaround is to change the `rootDeviceHints` to `minSizeGigabytes: 300`.
+
+After changing the `rootDeviceHints`, boot the CoreOS and then verify the disk information by using the command:
```

Comment:
```suggestion
After you change the `rootDeviceHints`, start the CoreOS and then verify the disk information by using the following command:
```
- IBM Style recommends "start" instead of "boot" where possible.
- By including the word "you", you can better engage the reader.